### DISCUSSION TITLE
Please Leave Your Contact

### DISCUSSION THREAD

**Discussion by raccoman** [kingToolbox/WindTerm#323] [General]
Hi, i would like to make to you a proposal, can you leave me any contact info? LinkedIn, Discord or email ecc..

Thanks.

**Comment by kingToolbox** [comment]
Of course, you can send emails to my mailbox `WindServices@yandex.com` and I am also looking forward to hearing from you. But I will leave for vacation soon. There is no Internet during the period. It may take many days to reply to your email, please understand, thank you!

**Reply by raccoman** [reply]
Did you get the email?
